mlperf.conf

```diff
@@ -28,28 +30,33 @@ stable-diffusion-xl.*.performance_sample_count_override = 5000
 
 *.SingleStream.target_latency_percentile = 90
 *.SingleStream.min_duration = 600000
-#*.SingleStream.min_query_count = 1024
 
 *.MultiStream.target_latency_percentile = 99
 *.MultiStream.samples_per_query = 8
 *.MultiStream.min_duration = 600000
-#*.MultiStream.min_query_count = 270336
 *.MultiStream.min_query_count = 662
 retinanet.MultiStream.target_latency = 528
 
+# With early stopping enabled, the min_query_count is 1 for Server Scenario
+*.Server.min_query_count = 1
+*.SingleStream.min_query_count = 100
+*.MultiStream.min_query_count = 662
 
-# 3D-UNet uses equal issue mode
+# 3D-UNet uses equal issue mode because it has non-uniform inputs
 3d-unet.*.sample_concatenate_permutation = 1
 
-# GPT-J uses equal issue mode for Single-Stream
-gptj.SingleStream.sample_concatenate_permutation = 1
+# LLM benchmarks have non-uniform inputs and outputs, and use equal issue mode for all latency scenario
 gptj.Server.sample_concatenate_permutation = 1
+gptj.SingleStream.sample_concatenate_permutation = 1
+gptj.MultiStream.sample_concatenate_permutation = 1
```

Comment:
No multistream for gptj - this line can be removed right?

Reply:
Removed.